library/src/main/java/com/parse/FacebookController.java

```diff
@@ -139,9 +139,9 @@ public void onError(FacebookException e) {
     return authData;
   }
 
-  public void setAuthData(Map<String, String> authData)
+  public void setAuthData(Map<String, String> authData, boolean isFacebookLoginScopeExternallyManaged)
       throws java.text.ParseException {
-    if (authData == null) {
+    if (authData == null && !isFacebookLoginScopeExternallyManaged) {
```

Comment:
I'll check regarding the iOS repo, but authData==null occurs on a logout flow (which is triggered from a login flow).